examples/abstract-objects/abstract-objects.bal

```diff
@@ -0,0 +1,57 @@
+import ballerina/io;
+
+// Defines an abstract object called 'Person'. It describes the type of
+// each field and method. However it cannot have a constructor method,
+// or any attached functions.
```

Comment:
Minor changes:
"// Define an abstract object called 'Person'. Be sure to describe
// each field type and method. Note that an abstract object cannot have a constructor method
// or any attached functions."